docs/upgrade/UPGRADE-v8.1.0-dev.md

```diff
@@ -192,14 +192,14 @@ There you can find links to upgrade notes for other versions too.
 
       <import file="${path.framework}/build.xml"/>
     ```
-- increase your PHPStan level to 4 in your `build.xml` ([#1040](https://github.com/shopsys/shopsys/pull/1040))
+- increase your PHPStan level to 4 in your `build.xml` and address all the reported violations ([#1381](https://github.com/shopsys/shopsys/pull/1381))
     ```diff
   - <property name="phpstan.level" value="1"/>
   + <property name="phpstan.level" value="4"/>
     ```
     - a lot of the possible issues should be already resolved if you followed the previous instruction and ran the `php phing annotations-fix` phing command
     - some of the issues related to class extension need to be addressed manually nevertheless (see the ["Framework extensibility" article](../introduction/framework-extensibility.md#problem-3)) for more information
-    - you need to resolve all the other reported problems (any of them can be ignored in your `phpstan.neon`). You can find inspiration in [#1040](https://github.com/shopsys/shopsys/pull/1040)
+    - you need to resolve all the other reported problems (it is up to you whether you decide to address them directly or add ignores in your `phpstan.neon`). You can find inspiration in [#1381](https://github.com/shopsys/shopsys/pull/1381)
```

Comment:
Why did you remove mention of #1040 ? I thought that some inspiration can be also found there, especially for fixing your own features...

Reply:
Because I did not see any added value in linking the old PR. All the current issues with phpstan were addressed in this opened PR (however the truth is I was looking for inspiration #1040)

Reply:
My thought is that when implementators want to fix stuff in their features they could use our fixes from #1040 to see what type of errors occurred and how to fix them...

Reply:
Yeah that might help as well...
```suggestion
    - you need to resolve all the other reported problems (it is up to you whether you decide to address them directly or add ignores in your `phpstan.neon`). You can find inspiration in [#1381](https://github.com/shopsys/shopsys/pull/1381) and [#1040](https://github.com/shopsys/shopsys/pull/1040)
```